hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -372,7 +369,14 @@ As with earlier Raspberry Pi models there is no method for specifying the boot d
 
 N.B. config.txt [conditional filters](../configuration/config-txt/conditional.md) can be used to select alternate firmware in complex device configurations.
 
-### Update the bootloader
+### Updating the bootloader
+#### Update using the Raspberry Pi Imager
+The easiest method to update the bootloader with a factory default configuration supporting USB boot is to use the Raspberry Pi Imager.
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)
+1. Download the latest [EEPROM recovery image](https://github.com/raspberrypi/rpi-eeprom/blob/master/releases.md)
+1. Select `Use Custom` to flash an SD card with the EEPROM update enabling USB MSD boot support.
```

Comment:
Maybe mention that this will erase whatever was previously on the SD card, so this should probably be done with a separate SD card, rather than the card the user has been running Raspberry Pi OS from?

Reply:
added *blank*

Reply:
The Raspberry Pi Imager does warn you though